### PR TITLE
Manipulator node

### PR DESCRIPTION
**Pull request by mortenfyhn**

### CODE REVIEW

File: manipulator_interface/scripts/manipulator_interface.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+        self.sub = rospy.Subscriber('manipulator_command', Manipulator, self.callback)
+
+        # TODO(mortenfyhn): Consider setting neutral to fully open instead
+        self.neutral_pulse_width = self.microsecs_to_bits(self.servo_position_to_microsecs(0))
+
+        self.valve_stepper = Stepper(STEPPER_NUM_STEPS, STEPPER_VALVE_PINS)
+        self.valve_stepper.set_speed(STEPPER_VALVE_RPM)
+        self.valve_direction = 0
+
+        rospy.sleep(0.1) # Initial set to zero seems to disappear without a short sleep here
```
Comment (fonstein) [inline manipulator_interface/scripts/manipulator_interface.py]: wat

File: vortex_msgs/msg/Manipulator.msg [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+# Normalized claw position
+# Must be in the range [-1, 1]
+float64 claw_position
+
+# Valve spinning
+# Set either true for corresponding action
+# Do not set both true
+bool open_valve
+bool close_valve
```
Comment (fonstein) [inline vortex_msgs/msg/Manipulator.msg]: This is wrong. Fix plz.

**Review by fonstein — CHANGES_REQUESTED** [review]

File: manipulator_interface/scripts/manipulator_interface.py (hunk 1 above)
Comment (mortenfyhn) [reply manipulator_interface/scripts/manipulator_interface.py]: ¯\_(ツ)_/¯

Comment (sindrehan) [reply manipulator_interface/scripts/manipulator_interface.py]: You dropped this:  \

File: vortex_msgs/msg/Manipulator.msg (hunk 2 above)
Comment (sindrehan) [reply vortex_msgs/msg/Manipulator.msg]: I will fix this.

File: manipulator_interface/scripts/stepper.py [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+                steps_left -= 1
+
+                self.step_motor(self.curr_step % 4)
+
+    def step_motor(self, curr_step):
+        """"Move motor forward or backward (low-level command)."""
+        for i in range(len(self.pins)):
+            pin_value = STEPPER_PIN_VALUES[curr_step][i]
+            if pin_value == 1:
+                GPIO.output(pin[i], GPIO.HIGH)
```
Comment (sindrehan) [inline manipulator_interface/scripts/stepper.py]: Should `pin` be `self.pins`? Can't find the declaration of `pin`. Same goes for line 80.

Comment (mortenfyhn) [reply manipulator_interface/scripts/stepper.py]: Yes. I fix.

**Review by fonstein — APPROVED** [review]

**Pull request merged**